www/js/pages/profile/profile.controller.js

```diff
@@ -67,10 +67,12 @@
 
         vm.init = () => {
             vm.isVirtualAccount = !!appStateService.virtuality;
+            vm.account = accountService.getDefault();
             if (!vm.isVirtualAccount) {
-                vm.account = accountService.getDefault();
                 vm.isFinancial = /MF/i.test(vm.account.id);
                 websocketService.sendRequestFor.residenceListSend();
+            } else if (vm.isVirtualAccount && !vm.account.country) {
+                websocketService.sendRequestFor.residenceListSend();
             } else {
```

Comment:
@nazaninreihani
since the value of `vm.isVirtualAccount` is a boolean so there isn't any need for the else clause. this part of the code is unreachable.

Reply:
@morteza-binary the else clause is for virtual accounts which have set country during signup, they do not pass first and second condition